web/app/routes/authenticated/document.js

```diff
@@ -106,10 +105,13 @@ export default class DocumentRoute extends Route {
       console.log("Error recording analytics: " + err);
     }
 
-
-
-    // Record the doc with the RecentlyViewedDocs service.
-    void this.recentDocs.markViewed.perform(params.document_id, params.draft);
+    // Update recently viewed docs for the dashboard route.
+    try {
+      await this.recentDocs.fetchAll.perform();
+    } catch (e) {
+      console.error("error updating recently viewed docs", e);
+      throw e;
+    }
```

Comment:
This doesn't seem important here. What if you remove it and defer that load for when the user is actually visiting the dashboard? Otherwise I'd call it with `void` instead of `try { await }` so it doesn't hold up loading other stuff.

Reply:
I was just minimizing changes, and when I fully removed this, the recently viewed docs weren't being updated in the dashboard until I refreshed the page. I just tried moving this to the dashboard route though and it seems to work, so I can move it there.

Reply:
https://github.com/hashicorp-forge/hermes/pull/144/commits/8143ed4ec9ae99a5439578ce3cf49ec30c6020fb